app/controllers/playback_controller.rb

```diff
@@ -16,6 +19,8 @@ def play
       # If recording isn't protected, don't check tokens
       deliver_resource(true)
     end
+  rescue RecordingNotFoundError
```

Comment:
Rather than have the `rescue` inside this controller method, I think it would be better to  a `rescue_from` handler at the top controller level: see https://guides.rubyonrails.org/action_controller_overview.html#rescue-from